### PR TITLE
this is sidebar

### PR DESCRIPTION
**Pull request by fayez-baig**

### CODE REVIEW

**Review by coder-mezab — APPROVED** [review]

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]
code looks good , though there is nothing

**Review by nasir-k — APPROVED** [review]

**Comment by shaikh7abish** [comment]
Grammatical mistakes

**Review by Coder-Arfat — COMMENTED** [review]
this is a grammar mistake

**Review by Naufil-Shemle — COMMENTED** [review]
Nice

**Review by faran4engg — COMMENTED** [review]
galat h bhai

File: sidebar.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is sidebar a
```
Comment (faran4engg) [inline sidebar.txt:1]: ## update
this should be `This is a sidebar !`

File: sidebar.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+This is a sidebar
```
Comment (Tauqeer-Ahmed-99) [inline sidebar.txt]: convent school se pdha hai na fayez kya hai bhai  a ko pehle likh

**Review by faran4engg — APPROVED** [review]

**Review by AbbasShaikh0304 — APPROVED** [review]
THis soluld be  This is sidebar!

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]

**Review by Tanvir04khan — APPROVED** [review]

**Pull request merged**